src/mono/mono/mini/interp/interp.c

```diff
@@ -2884,45 +2884,57 @@ init_arglist (InterpFrame *frame, MonoMethodSignature *sig, stackval *sp, char *
  * this/static * ret/void * 16 arguments -> 64 functions.
  */
 
+#if HOST_BROWSER
+#define INTERP_ENTRY_UPDATE_HIT_COUNT(_method) \
+	mono_interp_record_interp_entry (_method)
+#else
```

Comment:
Will this call into js for every interp entry ?

Reply:
Only until it gets jitted, I can move some of this tracking into C if necessary

Reply:
It should have some kind of fastpath like if (jiterpreter_enabled) etc. Also, shouldn't this be in interp_entry () ?

Reply:
Putting it in the generic entry is bad since those never get jitted, no point. Will put in a fast path, good point